environments/configuration.yml

```diff
@@ -65,6 +65,11 @@ required_packages_extra:
 # with serial = 1.
 osism_serial_default: 100%
 
+osism_serial:
+  # The recommended default for a production is 1 (is set by default).
+  # It is here 100% to speed up the Testbed.
+  network: 100%
```

Comment:
I don't understand why we need both `osism_serial_default` and this, shouldn't one override for everything be enough?

Reply:
At the moment we have `serial: "{{ osism_serial['network']|default('1') }}"` in the network playbook to have 1 as default when osism_serial[network] is not set. Same for osism_serial[docker].  For all other plays we use `osism_serial_default` as default (and 0 when `osism_serial_default` is not set).